packages/toast/src/ToastContainer/utils/useToastTransitions.ts

```diff
@@ -60,6 +60,13 @@ export function useToastTransitions({
     [exitCallback, getShouldExpand],
   );
 
+  useEffect(() => {
+    return () => {
+      handleTransitionEnter.cancel();
+      handleTransitionExit.cancel();
+    };
+  }, [handleTransitionEnter, handleTransitionExit]);
```

Comment:
Hey! Sorry for the random comment on a PR that was already merged a month ago, but won't this re-run on every render? As it stands right now, `enterCallback` and `exitCallback` aren't wrapped in `useCallback` in `ToastContainer`, which means both of `handleTransitionEnter` and `handleTransitionExit` will also be redefined on every render.

Reply:
Great catch! Opened a bug ticket to address this, thanks! [LG-4531](https://jira.mongodb.org/browse/LG-4531)